app/Actions/Service/ProbeServices.php

```
@@ -0,0 +1,255 @@
+            $descriptors[] = [
+                'service' => $service,
+                'handler' => $handler,
+                'unit' => $unit,
+                'versionCommand' => $versionCommand,
+                'networkingCommand' => $networkingCommand,
+                'networkingRequiresRunning' => $networkingRequiresRunning,
+            ];
+        }
+
+        return $descriptors;
+    }
```

PHPMD (2.15.0)

[warning] The method descriptors() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)